### ISSUE TITLE
GoogleSheetsAuthentication

### ISSUE THREAD

**Issue by ddshep** [sbstusa/template-reanalysis#3]
If I fail to authenticate when pulling from google sheets, it won't prompt again for authentication. How to reset and re-run auth? I'll try to look later.

**Comment by jwbowers** [comment]
Hmm... I don't know when it times out. Perhaps a question for the googlesheets maintainer?  Of course, the example document that I made depends on your ability to access our team .gov Google Drive so you would not be able to authenticate. 

I'd love to be able to use a key or something so that I didn't have to run the file in interactive mode for the first time in X time.

**Comment by ddshep** [comment]
When I have a moment I'll reach out to the maintainer (do you know them personally?) and see what I can find out. While in process I hit enter without pasting and then it didn't let me try again. Even after clearing and reopening a new project. 

I wonder for a key if we might ask the AWS security head at GSA. Do you want me to ask him as well on that point? If it can detect our SSO or something like that.

**Comment by jwbowers** [comment]
I don’t know the maintainer. I suspect that the first issue with keys will be with the googlesheets package itself. If it can handle them. At that point, we can think about how to generate them.
